tests/plotting/test_plotting.py

```diff
@@ -934,12 +934,19 @@ def test_add_arrows():
 
 def test_axes():
     plotter = pyvista.Plotter()
-    plotter.add_orientation_widget(pyvista.Cube())
+    plotter.add_orientation_widget(pyvista.Cube(), color='b')
     plotter.add_mesh(pyvista.Cube())
     plotter.show(before_close_callback=verify_cache_image)
 
 
 def test_box_axes():
+    plotter = pyvista.Plotter()
+    plotter.add_axes(box=True)
+    plotter.add_mesh(pyvista.Sphere())
+    plotter.show(before_close_callback=verify_cache_image)
+
+
+def test_box_axes_color_box():
     plotter = pyvista.Plotter()
     plotter.add_axes(box=True, box_args={'color_box': True})
     plotter.add_mesh(pyvista.Sphere())
```

Comment:
If I understand correctly, the new test was there to cover https://github.com/pyvista/pyvista/blob/8379d4a4b4a6de5cf7c9cc9cf4545d2be428aee6/pyvista/plotting/tools.py#L105-L106

It turns out that this is tripped if we pass `box=True` without `color_box=True` in the `box_args` dict. So what I did was rename the old test to `test_box_axes_color_box()` and add a new test by the old name (`test_box_axes()`) that doesn't enable colouring for the axes orientation widget. This fixes coverage of that line for me locally. I've also pushed both cache images.

Do let me know if I misunderstood the premise here.

Reply:
No, that's perfect!